craco.config.js

```diff
@@ -10,8 +10,8 @@ module.exports = {
             paths.appIndexJs,
           ].filter(Boolean),
           content: [
-            "./src/chromeServices/content.ts",
-            "./src/chromeServices/DOMEvaluator.ts",
+            "./src/background/content.ts",
+            "./src/services/chromeServices/DOMEvaluator.ts",
           ],
```

Comment:
are we going to have more services than chromeServices/domevaluator? if not maybe just moving domevaluator into services could be better

Reply:
Done.